cmd/outofband.go

```diff
@@ -2,6 +2,7 @@ package cmd
 
 import (
 	"flag"
+	"math/rand"
```

Comment:
why not use `crypto/rand` instead? I was gonna comment about this anyway even if I hadn't seen the gosec ignore statement that was written below.